firestore/angular-rewrite/src/app/app.module.ts

```diff
@@ -72,7 +73,11 @@ import { MatButtonModule } from '@angular/material/button';
     BrowserAnimationsModule,
     provideFirebaseApp(() => initializeApp(environment.firebase)),
     provideAuth(() => getAuth()),
-    provideFirestore(() => getFirestore()),
+    provideFirestore(() => {
+      const firestore = getFirestore();
+      connectFirestoreEmulator(firestore, 'localhost', 8080);
```

Comment:
nit: check environment and decide whether to call based on some external configuration

Reply:
I think using environments.ts would work here - what we will want to do in the future is allow devleopers to add their prod config and deploy this on firebase hosting. Ideally this will be all done in one or two places, consider:

1. Add your prod config to environments.ts
2. Set your selected config to "prod" instead of "local"

-> boot up the App and now instead of connecting to the emulators it connects to your prod environment

Reply:
Gotcha; In an upcoming commit, I've modified `environments.ts` to allow for a production config and added a check in `app.module.ts` that checks if the `demo` or `production` version is running.